### ISSUE TITLE
Not working with Svelte

### ISSUE THREAD

**Issue by soullivaneuh** [tonai/storybook-addon-themes#15]
Using Storybook 5.3.6 with this Svelte component story:

```javascript
import { action } from '@storybook/addon-actions';
import {
  text, boolean, select,
} from '@storybook/addon-knobs';
import { withThemes } from 'storybook-addon-themes';
import Container from '../views/Container.svelte';
import Button from './Button.svelte';
import ButtonView from '../views/ButtonView.svelte';

export default {
  title: 'Button',
  component: Button,
  decorators: [
    withThemes
  ],
  parameters: {
    notes: 'This is a button. You can click on it.',
  },
};

export const Default = () => ({
  Component: Container,
  props: {
    component: ButtonView,
    props: {
      text: text('text', 'Click me!'),
      button: {
        href: text('href'),
        type: select('type', ['button', 'submit']),
        icon: text('icon'),
        outline: boolean('outline'),
        color: select('color', ['primary', 'secondary', 'danger']),
        disabled: boolean('disabled'),
      },
    },
  },
  on: {
    click: action('click'),
  },
});
```

I have the following error:

```
Expecting a Svelte component from the story: "Default" of "Input".

Did you forget to return the Svelte component configuration from the story?
Use "() => ({ Component: YourComponent, data: {} })"
when defining the story.
```

If I remove `withThemes`, everything works fine.

Note: I added the parameters on global `config.js` file:

```javascript
import { addParameters } from '@storybook/svelte';

addParameters({
  themes: [
    { name: 'twitter', class: 'theme-twt', color: '#00aced', default: true },
    { name: 'facebook', class: 'theme-fb', color: '#3b5998' },
  ],
});
```

But this change not a thing even if I remove it.

**Comment by soullivaneuh** [comment]
I gave a try with latest `5.3.9` version of storybook, same issue.

**Comment by tonai** [comment]
Hi @soullivaneuh ,
The decorator version only works with React and Vue for now.
Take a look [here](https://github.com/tonai/storybook-addon-themes#framework-support-table).
I also have updated the README to make it clearer that the decorator version is not ready for all framework.

For Svelte you can use the [standard configuration](https://github.com/tonai/storybook-addon-themes#configuration).
I just tried it myself (using the global configuration) on a brand new project initialized with:
```bash
npx degit sveltejs/template storybook-svelte
cd storybook-svelte
npx -p @storybook/cli sb init --type svelte
```
and it works.
So tell me if that fit your needs.

**Comment by tonai** [comment]
Svelte decorator support has been released in version 6.0.0-alpha.1 that support the new storybook 6.x version.
